src/rdkafka_mock_cgrp.c

```diff
@@ -1402,19 +1402,65 @@ rd_kafka_mock_cgrp_consumer_member_find_by_instance_id(
 static rd_bool_t rd_kafka_mock_cgrp_consumer_member_subscribed_topic_names_set(
     rd_kafka_mock_cgrp_consumer_member_t *member,
     rd_kafkap_str_t *SubscribedTopicNames,
-    int32_t SubscribedTopicNamesCnt) {
-        rd_bool_t changed = rd_false;
-        if (!SubscribedTopicNames)
+    int32_t SubscribedTopicNamesCnt,
+    const rd_kafkap_str_t *SubscribedTopicRegex) {
+        rd_bool_t changed           = rd_false;
+        rd_list_t *new_subscription = NULL;
+        int32_t i;
+
+        if (!SubscribedTopicNames &&
+            RD_KAFKAP_STR_IS_NULL(SubscribedTopicRegex) &&
+            !member->subscribed_topic_regex)
```

Comment:
Here it should return changed also if `subscribed_topic_regex` is an empty string

Reply:
After this `if` we can have an assert that both values must be non-NULL and either one or the other must be non-empty. We ensure this in the subscribe call, we return invalid `_INVALID_ARG` if passing an empty list or if it's containing invalid regexes